src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/signature_pending.jelly

```diff
@@ -0,0 +1,70 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <div class="custom-pane-frame-padding">
+        <j:choose>
+            <j:when test="${it.pendingSignatures.isEmpty()}">
+                <p>
+                    No pending signature approvals.
+                </p>
+            </j:when>
+            <j:otherwise>
+                <j:forEach var="s" items="${it.pendingSignatures}">
+                    <div id="s-${s.hash}">
+                        <p>
+                            <button onclick="approveSignature('${s.signature}', '${s.hash}')">Approve</button> /
+                            <j:if test="${!s.signature.startsWith('field')}">
+                                <button onclick="aclApproveSignature('${s.signature}', '${s.hash}')">Approve assuming permission check</button> /
+                            </j:if>
+                            <button onclick="denySignature('${s.signature}', '${s.hash}')">Deny</button> signature
```

Comment:
```suggestion
                            <span class="yui-button"><button onclick="denySignature('${s.signature}', '${s.hash}')">Deny</button></span> signature
```

Reply:
signature_pending = only moved, nothing intented to be changed